src/js/actions/index.js

```diff
@@ -66,6 +66,8 @@ define(function (require, exports, module) {
         transform: require("./transform"),
         type: require("./type"),
         typetool: require("./typetool"),
-        ui: require("./ui")
+        ui: require("./ui"),
+        verifyDocuments: require("./verify/documents.js"),
```

Comment:
Not needed in this PR, but it might be useful to have a single index file for verification methods that we can exclude in release builds.